scripts/travis/build.sh

```diff
@@ -65,6 +65,17 @@ else
   pullUpstreamTag "${COM_UPSTREAM_REPO}" "${COM_DEPENDENCY_VERSION}"
 fi
 
```

Comment:
I'm not sure this will work with HFs where there is a label too (generally edited only in the enterprise repo). You would need to see what the CX team do with their WoW. 

This check would only really come into its own when creating new branches. I don't think I have ever forgotten to change the acs-packaging version.

Reply:
The check will work fine with labels or HF branches. It just checks that the acs-packaging version starts with `[major].[minor].[revision]` where those variables come from alfresco-community-repo. You're right that the check is only useful when updating those three digits of the release.